src/components/Modal/Modal.styled.jsx

```diff
@@ -62,8 +62,9 @@ export const InfoContainer = styled.div`
   }
 
   h4{
-    margin-top: 0;
-    margin-bottom: 2rem;
+    margin-top: 5px;
+    margin-bottom: 1.8rem;
+    font-weight: normal;
```

Comment:
In another views the title is `bold` so I recomend you create a new componente to title and avoid change the global styles